plugins/emoticons.md

```diff
@@ -9,20 +9,24 @@ controls: toolbar button
 
 This plugin adds a dialog to the editor lets users insert emojis into {{site.productname}}'s editable area. The dialog can be invoked via a toolbar button - `emoticons` - or a dedicated menu item added as `Insert > Emoticons`.
 
-> Note: it doesn't automatically convert text emoticons into graphical smilies.
+The emoticons plugin provides an autocompleter for adding emoji without using the toolbar button or menu item. Adding a colon `:`, followed by at least two characters will open the emoticon picker showing matching emoji.
```

Comment:
Should `emoji` be plural here?

Reply:
the plural of emoji can be either emoji or emojis. though line 10 conflicts at the moment...

Reply:
Yeah, that is what made me comment. I personally, don't have a preference as long as it consistent.

Reply:
s/emojis/emoji/g excluding codeblocks